batect.yml

```diff
@@ -21,7 +21,7 @@ config_variables:
   gradle_build_scan:
     description: Enabling of Gradle build scans
     # Supported values: "true", "false"
-    default: false
+    default: "false"
```

Comment:
Propose to move the change to batect.yml to a separate commit something like "batect: Quote boolean string"

Reply:
Is this change even necessary?